### PR TITLE
tailwind configuration impl added

### PR DESCRIPTION
**Pull request by matt-7t**
Code updated to expose custom tailwind configuration as an integrant component that can be initialized with custom styles

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -92,3 +92,6 @@ Temporary Items
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+
+.clj-kondo/
```
Comment (matt-7t) [inline .gitignore:96]: Us poor VSCode users :p

File: src/tailwind/core.cljs [hunk 2]
```diff
@@ -8,20 +8,42 @@
...
 
-(def ^:private tw-rn
-  (when (exists? js/require) (js/require "tailwind-rn")))
+(declare created tw-rn colors)
+
+(defmethod ig/init-key :tailwind/config
+  [_ {:keys [styles]}]
+  (reset! created (j/call tw-rn :create (clj->js styles))))
+
+(defmethod ig/halt-key! :tailwind/config
```
Comment (sandhu) [inline src/tailwind/core.cljs:21]: You can simply skip the halt-key! if there is no teardown code.

File: project.clj [hunk 3]
```diff
@@ -12,6 +12,7 @@
   :url "https://github.com/7theta/tailwind"
   :license {:name "MIT License"
             :url "https://opensource.org/licenses/MIT"}
```
Comment (sandhu) [inline project.clj:14]: Please also upversion the project to `0.2.0`

**Review by sandhu — CHANGES_REQUESTED** [review]

File: project.clj [hunk 4]
```diff
@@ -12,6 +12,7 @@
...
-  :dependencies [[com.7theta/utilis "1.12.0"]]
+  :dependencies [[com.7theta/utilis "1.12.0"]
```
Comment (sandhu) [inline project.clj:15]: Please upversion to `1.12.1`

**Review by sandhu — APPROVED** [review]

**Pull request merged**